package.json

```diff
@@ -29,14 +31,14 @@
     "lint:sass": "yarn stylelint '{public/sass,packages}/**/*.scss' --cache",
     "test:ci": "mkdir -p reports/junit && JEST_JUNIT_OUTPUT_DIR=reports/junit jest --ci --reporters=default --reporters=jest-junit -w ${TEST_MAX_WORKERS:-100%}",
     "lint:fix": "yarn lint:ts --fix",
-    "packages:build": "nx run-many -t build --projects='@grafana/*'",
+    "packages:build": "nx run-many -t build --projects='@grafana/*,@test-plugins/*'",
```

Comment:
Hmm... I'm not very keen on introducing this here as it means any CI pipeline that runs this script will also build all test plugins which could introduce failures during CI and/or during a release. Could you explain why it's necessary?

Reply:
No mostly me not being lazy and not wanting to fiddle with the drone config. 

I've [changed this](https://github.com/grafana/grafana/pull/91873/commits/c8fb693aac139c345fa91deebdc579878c673d4a) so that test plugins are now being built in a separate build step. Let me know what you think.